app/uk/gov/hmrc/slacknotifications/services/AuthService.scala

```diff
@@ -58,6 +59,37 @@ class AuthService @Inject()(configuration: Configuration) {
     } else {
       true
     }
+
+  def isAuthorizedUrl(url: String): Boolean = authConfiguration.authorizedUrls.contains(url)
+
+  def filterFieldsForURLs(fields: Array[String]): Array[String] = {
+    val pattern =
+      ("(?:[a-zA-Z]+:)?(?:\\/\\/)?(?:[\\w-]+:[\\w-]+@)?(?:[\\w-]+\\.)+[a-zA-Z]+" +
```

Comment:
it's virtually impossible to tell what this regex checks, maybe the variable name can provide hints?

Reply:
Done :+1: